dcos_launch/platforms/onprem.py

```diff
@@ -96,6 +96,9 @@ def install_dcos(
         check_results(
             all_client.run_command('run', ['chmod +x ~/install_prereqs.sh', '&&', '~/install_prereqs.sh']), node_client,
             'install DC/OS prerequisites')
+        setenforce = '1' if enable_selinux else '0'
+        check_results(all_client.run_command('run', ['sudo setenforce ' + setenforce]), node_client,
+                      'Enable SELinux in enforcing mode')
```

Comment:
Everything else looks great and this is a very minor and possible inapplicable nit:
Would 'Enable SELinux in enforcing mode' be an applicable tag if enable_selinux was 0? or could the message be adapted a bit?

Reply:
Yeah, a better message will be less confusing. (Or just the print the command and let the user understand what is happening instead of printing this help message).